galley/pkg/config/analysis/analyzers/analyzers_test.go

```diff
@@ -68,6 +68,80 @@ var testGrid = []testCase{
 			{msg.MisplacedAnnotation, "Namespace staging"},
 		},
 	},
+
```

Comment:
Nittiest nit: Extra space crept in here.

Reply:
Oops, thanks. Fixed.